S21/hvkone/refactorme.py

```diff
@@ -2,83 +2,57 @@
 
 class Word:
 
-  def __init__(self, word, bcpCode, std=False):
-    self._w = word
-    self._l = bcpCode
-    self._finalSigma = False
-    self._standardIrishSpelling = std
-    # OLD EXPERIMENTAL CODE for dealing with vowel harmony
-    # self._numVowels = 0
-    # for c in word:
-    #   if c in 'aeiouAEIOU':
-    #   self._numVowels += 1
+  def __init__(self, word, language):
+    self.word = word 
+    self.lang = language
+  
+  def get_language(self):
+    language = self.lang
+    
+    if '-' in self.lang:
+      index = self.lang.find('-')
+      language = self.lang[0:index]
+      return language
+      
+    if not len(language)==2:
+      raise ValueError("Invalid BCP-47 code")
+    return language
+    
+  def change_to_lower(self):
+    language=self.get_language()
+    not_lower_list = ('zh','ja','th')
+    Irish_vowels = ('AEIOU\u00c1\u00c9\u00cd\u00d3\u00da')
+    
+    if language in not_lower_list:
+      return self.word
 
-  def setWord(self, w):
-    self._w = w
+    elif language==('ga' or 'gd'):
```

Comment:
syntax is wrong here; this is checking if language equals-equals the boolean on the RHS.  Did you run tests?